src/PrestaShopBundle/Form/Admin/Sell/Discount/DiscountInformationType.php

```diff
@@ -51,7 +52,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             ])
             ->add('names', TranslatableType::class, [
                 'label' => $this->trans('Discount Name', 'Admin.Catalog.Feature'),
-                'label_help_box' => $this->trans('This will be displayed in the cart summary, as well as on the invoice.', 'Admin.Catalog.Help'),
+                'label_help_box' => $this->trans('For your eyes only. This will be displayed in the cart summary, as well as on the invoice.', 'Admin.Catalog.Help'),
```

Comment:
`For your eyes only.` is not true - the discount name is public. :-)

Description is the private one.